app/forms/supplementary/RepresentativeDetails.scala

```diff
@@ -49,7 +49,7 @@ object RepresentativeDetails {
   implicit val format = Json.format[RepresentativeDetails]
 
   private val representativeStatusCodeAllowedValues =
-    Set(StatusCodes.DirectRepresentative, StatusCodes.IndirectRepresentative)
+    Set(StatusCodes.Declarant,StatusCodes.DirectRepresentative, StatusCodes.IndirectRepresentative)
```

Comment:
Inside this object, you can import `StatusCodes` model
`import StatusCodes._` and after that, your Set can look like `Set(Declarant, DirectRepresentative, IndirectRepresentative)`, but it's not a problem